etc/profile-a-l/evince.profile

```diff
@@ -6,6 +6,10 @@ include evince.local
 # Persistent global definitions
 include globals.local
 
+# Uncomment this line and the bottom ones to use bookmarks
+# NOTE: This possibly exposes information, including file history from other programs.
```

Comment:
The allowed D-Bus names below
- allow editing of all metadata and (u)mounting (if I'm correct)
- maybe allow filesystem access outside of the sandbox (=escape)